### ISSUE TITLE
Minimum Node.js LTS versions: 18.20, 20.10, or 22

### ISSUE THREAD

**Issue by Jstrorma** [thoukydides/homebridge-homeconnect#300]
### Description of Issue

[homebridge-homeconnect] file:///volume1/homebridge/node_modules/homebridge-homeconnect/dist/settings.js:3
import PACKAGE from '../package.json' with { type: 'json' };
                                      ^^^^

SyntaxError: Unexpected token 'with'
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:118:18)
    at callTranslator (node:internal/modules/esm/loader:273:14)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:278:30)

Node.js v20.9.0

### Expected Behaviour

Open the settings

### Steps to Reproduce

Open the settings

### Plugin Version

1.0.4

### Environment

- OS: Synology NAS
- Node:
- Homebridge: 1.8.4

### Diagnostic Checks

- [ ] Home Connect app is able to control the appliance with Wi-Fi disabled.
- [ ] Home Connect app shows three green lines in the Network section of the appliance Settings.
- [ ] Unofficial [Home Connect Server Status](https://homeconnect.thouky.co.uk) shows servers up and no recent errors.
- [ ] Searched for [previously reported issues](https://github.com/thoukydides/homebridge-homeconnect/issues?q=is%3Aissue) and no similar ones found.
- [ ] Checked the [Home Connect API errors](https://api-docs.home-connect.com/general?#api-errors) and [authorisation errors](https://api-docs.home-connect.com/authorization?#authorization-errors) documentation.

### Log File

```shell
[homebridge-homeconnect] file:///volume1/homebridge/node_modules/homebridge-homeconnect/dist/settings.js:3
import PACKAGE from '../package.json' with { type: 'json' };
                                      ^^^^

SyntaxError: Unexpected token 'with'
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:118:18)
    at callTranslator (node:internal/modules/esm/loader:273:14)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:278:30)

Node.js v20.9.0
```

### Configuration

```json
{
    "platform": "HomeConnect",
    "name": "HomeConnect"
}
```

**Comment by thoukydides** [comment]
You require a more recent version of Node.js.

On the Node 20 branch you require at least 20.10.0 for nodejs/node#50140.

**Comment by Jstrorma** [comment]
Ok, that sucks. DSM only supports 20.9.0-1003 for now and I can't update the node.js from within HomeBridge using the hb-service command.

**Comment by thoukydides** [comment]
In that case you will need to use v1.0.3 or earlier of this plugin until Synology release an up-to-date version of Node.js. This plugin supports all Node.js LTS branches (18, 20, and the upcoming 22)… but does require a relatively recent version on each branch (at least 18.20.0 or 20.10.0).

This is due to a breaking change in Node.js 22 that was backported to the other LTS branches. Version v1.0.4 was updated for that change to fix #298.

You won’t be missing any new functionality in v1.0.4; the only change (other than updated dependencies) is the Node.js 22 compatibility. I don’t currently have any other enhancements planned until Apple update HAP (with the promised robot vacuum cleaner support).

**Comment by thoukydides** [comment]
[v1.0.5](https://github.com/thoukydides/homebridge-homeconnect/releases/tag/v1.0.5) should restore compatibility with Node.js 20.9.0.
